Dockerfile

```diff
@@ -31,10 +29,6 @@ RUN apk add --no-cache --virtual cluster-deps make musl-dev go git \
     && apk del --purge cluster-deps \
     && cd / && rm -rf /go/src /go/bin/gx /go/bin/gx-go
 
-USER ipfs
-
-VOLUME $IPFS_CLUSTER_PATH
-
 ENTRYPOINT ["/usr/local/bin/start-daemons.sh"]
```

Comment:
Can we try tini like this https://github.com/ipfs/go-ipfs/blob/master/Dockerfile#L65 ?